### ISSUE TITLE
Suggestion: Country Organization for Events

### ISSUE THREAD

**Issue by TechnologyClassroom** [phildini/stayinghomeclub#161]
It would be beneficial for conference decision makers to separate entries by country and date.  The page does not answer the question whether other conferences in my are are cancelling events.

Typical size of the conference would also be useful.

**Comment by koronkowy** [comment]
I agree that more organization functionality of the events list would make the list more beneficial.

Although, I don't know if the overhead of getting all of this information in to be able to submit an event and verifying it is a worthy cause. You can find out the information you're looking for by going to the individual sources that are listed by the event.

I also know from my experience in event organization that not all events publicly list the attendance numbers and/or do not have this information easily available. Many of these events in are in the future and they may not have closed their ticket sales before making the decision to postpone the event. Attendance numbers are more accurate post-event vs. pre-event unless there's a set defined number of tickets, and even then you don't know % sold unless the organization running the event chooses to be transparent.

**Comment by TechnologyClassroom** [comment]
Now that the data is in yaml instead of md, this issue is easier to work on without disrupting anything else.

**Comment by phildini** [comment]
I definitely approve of someone starting to tag the data this way.
